app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_like_counter
+
+  def update_like_counter
+    post.update(like_counter: post.likes.all.length)
+  end
```

Comment:
- [ ] [OPTIONAL] Good approach to solving this problem. I'd like to suggest a shorter and even more readable way to do this by using the increment method as shown below 👇
``` ruby
def update_like_counter
  post.increment!(:likes_counter)
end
```